web/src/pages/login.js

```diff
@@ -140,11 +144,20 @@ const Login = () => {
                     item className={classes.fullWidth}
                     xs={12}
                   >
-                    Need an account? <a href={'/register'} >Sign Up</a>
+                    Need an account?
```

Comment:
What's the difference between this, and using the built-in Link component? https://nextjs.org/docs/api-reference/next/link

Reply:
I believe Devin just didn't know about the Link component at the time. it has since been changed to use `Link`